Error when trying to install

<details>
<summary> Error when compiling (`cargo install godam`)</summary>

```
  Updating crates.io index
  Downloaded godam v0.1.1
  Downloaded 1 crate (27.6 KB) in 0.92s
  Installing godam v0.1.1
    Updating crates.io index
     Locking 213 packages to latest compatible versions
      Adding core-foundation v0.9.4 (latest: v0.10.0)
      Adding encode_unicode v0.3.6 (latest: v1.0.0)
      Adding foreign-types v0.3.2 (latest: v0.5.0)
      Adding foreign-types-shared v0.1.1 (latest: v0.3.1)
      Adding generic-array v0.14.7 (latest: v1.1.0)
      Adding hermit-abi v0.3.9 (latest: v0.4.0)
      Adding idna v0.5.0 (latest: v1.0.2)
      Adding linux-raw-sys v0.4.14 (latest: v0.6.5)
      Adding security-framework v2.11.1 (latest: v3.0.0)
      Adding unicode-width v0.1.14 (latest: v0.2.0)
      Adding wasi v0.11.0+wasi-snapshot-preview1 (latest: v0.13.2+wasi-0.2.1)
      Adding windows-registry v0.2.0 (latest: v0.3.0)
      Adding windows-strings v0.1.0 (latest: v0.2.0)
      Adding windows-sys v0.52.0 (latest: v0.59.0)
  Downloaded anstream v0.6.15
  Downloaded aes v0.8.4
  Downloaded anstyle v1.0.8
  Downloaded bumpalo v3.16.0
  Downloaded autocfg v1.4.0
  Downloaded atomic-waker v1.1.2
  Downloaded cc v1.1.22
  Downloaded cipher v0.4.4
  Downloaded bzip2 v0.4.4
  Downloaded bytes v1.7.2
  Downloaded bzip2-sys v0.1.11+1.0.8
  Downloaded clap v4.5.18
  Downloaded clap_builder v4.5.18
  Downloaded clap_lex v0.7.2
  Downloaded colorchoice v1.0.2
  Downloaded clap_derive v4.5.18
  Downloaded console v0.15.8
  Downloaded flate2 v1.0.34
  Downloaded crc-catalog v2.4.0
  Downloaded crc v3.2.1
  Downloaded deflate64 v0.1.9
  Downloaded http v1.1.0
  Downloaded hmac v0.12.1
  Downloaded http-body-util v0.1.2
  Downloaded http-body v1.0.1
  Downloaded h2 v0.4.6
  Downloaded hyper-rustls v0.27.3
  Downloaded indexmap v2.5.0
  Downloaded hyper-tls v0.6.0
  Downloaded hyper-util v0.1.9
  Downloaded indicatif v0.17.8
  Downloaded ipnet v2.10.0
  Downloaded is_terminal_polyfill v1.70.1
  Downloaded inout v0.1.3
  Downloaded lzma-rs v0.3.0
  Downloaded mio v1.0.2
  Downloaded native-tls v0.2.12
  Downloaded openssl v0.10.66
  Downloaded pbkdf2 v0.12.2
  Downloaded openssl-macros v0.1.1
  Downloaded openssl-sys v0.9.103
  Downloaded portable-atomic v1.9.0
  Downloaded reqwest v0.12.7
  Downloaded rustls-pemfile v2.1.3
  Downloaded rustls-pki-types v1.9.0
  Downloaded thiserror v1.0.64
  Downloaded serde_spanned v0.6.8
  Downloaded syn v2.0.79
  Downloaded tokio-macros v2.4.0
  Downloaded thiserror-impl v1.0.64
  Downloaded tokio-native-tls v0.3.1
  Downloaded sync_wrapper v1.0.1
  Downloaded toml v0.8.19
  Downloaded tower-service v0.3.3
  Downloaded toml_datetime v0.6.8
  Downloaded winnow v0.6.20
  Downloaded unicode-normalization v0.1.24
  Downloaded unicode-width v0.1.14
  Downloaded url v2.5.2
  Downloaded zerocopy-derive v0.7.35
  Downloaded zeroize_derive v1.4.2
  Downloaded zeroize v1.8.1
  Downloaded zopfli v0.8.1
  Downloaded zerocopy v0.7.35
  Downloaded zip v2.2.0
  Downloaded zstd-safe v7.2.1
  Downloaded anstyle-query v1.1.1
  Downloaded anstyle-parse v0.2.5
  Downloaded adler2 v2.0.0
  Downloaded cpufeatures v0.2.14
  Downloaded jobserver v0.1.32
  Downloaded miniz_oxide v0.8.0
  Downloaded subtle v2.6.1
  Downloaded serde_derive v1.0.210
  Downloaded idna v0.5.0
  Downloaded rustls-webpki v0.102.8
  Downloaded foreign-types-shared v0.1.1
  Downloaded foreign-types v0.3.2
  Downloaded constant_time_eq v0.3.1
  Downloaded hyper v1.4.1
  Downloaded lockfree-object-pool v0.1.6
  Downloaded libc v0.2.159
  Downloaded strsim v0.11.1
  Downloaded ppv-lite86 v0.2.20
  Downloaded pkg-config v0.3.31
  Downloaded openssl-probe v0.1.5
  Downloaded once_cell v1.20.1
  Downloaded number_prefix v0.4.0
  Downloaded tokio-rustls v0.26.0
  Downloaded tokio-util v0.7.12
  Downloaded toml_edit v0.22.22
  Downloaded zstd v0.13.2
  Downloaded utf8parse v0.2.2
  Downloaded zstd-sys v2.0.13+zstd.1.5.6
  Downloaded tokio v1.40.0
  Downloaded serde v1.0.210
  Downloaded serde_json v1.0.128
  Downloaded rustls v0.23.13
  Downloaded 98 crates (8.8 MB) in 3.75s
   Compiling libc v0.2.159
   Compiling proc-macro2 v1.0.86
   Compiling unicode-ident v1.0.13
   Compiling shlex v1.3.0
   Compiling pkg-config v0.3.31
   Compiling cfg-if v1.0.0
   Compiling typenum v1.17.0
   Compiling version_check v0.9.5
   Compiling once_cell v1.20.1
   Compiling pin-project-lite v0.2.14
   Compiling bytes v1.7.2
   Compiling vcpkg v0.2.15
   Compiling itoa v1.0.11
   Compiling futures-core v0.3.30
   Compiling serde v1.0.210
   Compiling equivalent v1.0.1
   Compiling hashbrown v0.14.5
   Compiling autocfg v1.4.0
   Compiling fnv v1.0.7
   Compiling tracing-core v0.1.32
   Compiling byteorder v1.5.0
   Compiling generic-array v0.14.7
   Compiling http v1.1.0
   Compiling futures-task v0.3.30
   Compiling log v0.4.22
   Compiling pin-utils v0.1.0
   Compiling slab v0.4.9
   Compiling httparse v1.9.4
   Compiling foreign-types-shared v0.1.1
   Compiling openssl v0.10.66
   Compiling quote v1.0.37
   Compiling futures-sink v0.3.30
   Compiling indexmap v2.5.0
   Compiling foreign-types v0.3.2
   Compiling futures-util v0.3.30
   Compiling tracing v0.1.40
   Compiling syn v2.0.79
   Compiling jobserver v0.1.32
   Compiling mio v1.0.2
   Compiling socket2 v0.5.7
   Compiling tinyvec_macros v0.1.1
   Compiling atomic-waker v1.1.2
   Compiling try-lock v0.2.5
   Compiling cc v1.1.22
   Compiling bitflags v2.6.0
   Compiling http-body v1.0.1
   Compiling native-tls v0.2.12
   Compiling subtle v2.6.1
   Compiling want v0.3.1
   Compiling tinyvec v1.8.0
   Compiling getrandom v0.2.15
   Compiling futures-channel v0.3.30
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.4
   Compiling utf8parse v0.2.2
   Compiling zstd-safe v7.2.1
   Compiling percent-encoding v2.3.1
   Compiling digest v0.10.7
   Compiling openssl-probe v0.1.5
   Compiling smallvec v1.13.2
   Compiling form_urlencoded v1.2.1
   Compiling anstyle-parse v0.2.5
   Compiling inout v0.1.3
   Compiling rand_core v0.6.4
   Compiling crc32fast v1.4.2
   Compiling powerfmt v0.2.0
   Compiling colorchoice v1.0.2
   Compiling is_terminal_polyfill v1.70.1
   Compiling unicode-bidi v0.3.15
   Compiling thiserror v1.0.64
   Compiling anstyle-query v1.1.1
   Compiling tower-service v0.3.3
   Compiling serde_json v1.0.128
   Compiling portable-atomic v1.9.0
   Compiling memchr v2.7.4
   Compiling unicode-normalization v0.1.24
   Compiling ryu v1.0.18
   Compiling crc-catalog v2.4.0
   Compiling anstyle v1.0.8
   Compiling cpufeatures v0.2.14
   Compiling adler2 v2.0.0
   Compiling crc v3.2.1
   Compiling http-body-util v0.1.2
   Compiling miniz_oxide v0.8.0
   Compiling anstream v0.6.15
   Compiling deranged v0.3.11
   Compiling cipher v0.4.4
   Compiling hmac v0.12.1
   Compiling idna v0.5.0
   Compiling semver v1.0.23
   Compiling base64 v0.22.1
   Compiling bumpalo v3.16.0
   Compiling openssl-sys v0.9.103
   Compiling zstd-sys v2.0.13+zstd.1.5.6
   Compiling bzip2-sys v0.1.11+1.0.8
   Compiling time-core v0.1.2
   Compiling simd-adler32 v0.3.7
   Compiling clap_lex v0.7.2
   Compiling heck v0.5.0
   Compiling lazy_static v1.5.0
   Compiling num-conv v0.1.0
   Compiling strsim v0.11.1
   Compiling unicode-width v0.1.14
   Compiling rustls-pki-types v1.9.0
   Compiling winnow v0.6.20
   Compiling zip v2.2.0
   Compiling lockfree-object-pool v0.1.6
   Compiling console v0.15.8
   Compiling clap_builder v4.5.18
   Compiling zopfli v0.8.1
   Compiling time v0.3.36
   Compiling rustls-pemfile v2.1.3
   Compiling bzip2 v0.4.4
   Compiling url v2.5.2
   Compiling flate2 v1.0.34
   Compiling aes v0.8.4
   Compiling pbkdf2 v0.12.2
   Compiling lzma-rs v0.3.0
   Compiling sha1 v0.10.6
   Compiling sync_wrapper v1.0.1
   Compiling encoding_rs v0.8.34
   Compiling mime v0.3.17
   Compiling constant_time_eq v0.3.1
   Compiling number_prefix v0.4.0
   Compiling ipnet v2.10.0
   Compiling deflate64 v0.1.9
   Compiling indicatif v0.17.8
   Compiling serde_derive v1.0.210
   Compiling tokio-macros v2.4.0
   Compiling openssl-macros v0.1.1
   Compiling zerocopy-derive v0.7.35
   Compiling thiserror-impl v1.0.64
   Compiling zeroize_derive v1.4.2
   Compiling tokio v1.40.0
   Compiling zerocopy v0.7.35
   Compiling zeroize v1.8.1
   Compiling clap_derive v4.5.18
   Compiling displaydoc v0.2.5
   Compiling ppv-lite86 v0.2.20
   Compiling rand_chacha v0.3.1
   Compiling rand v0.8.5
   Compiling clap v4.5.18
   Compiling tokio-util v0.7.12
   Compiling h2 v0.4.6
   Compiling tokio-native-tls v0.3.1
   Compiling serde_spanned v0.6.8
   Compiling toml_datetime v0.6.8
   Compiling serde_urlencoded v0.7.1
   Compiling toml_edit v0.22.22
   Compiling hyper v1.4.1
   Compiling toml v0.8.19
   Compiling hyper-util v0.1.9
   Compiling hyper-tls v0.6.0
   Compiling reqwest v0.12.7
   Compiling zstd v0.13.2
   Compiling godam v0.1.1
error[E0425]: cannot find function `exists` in module `std::fs`
  --> /home/als/.cargo/registry/src/index.crates.io-6f17d22bba15001f/godam-0.1.1/src/fs.rs:36:14
   |
36 |     std::fs::exists(path)
   |              ^^^^^^ not found in `std::fs`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `godam` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `godam v0.1.1`, intermediate artifacts can be found at `/tmp/cargo-installBXWaHa`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.

```

</details>

Cargo Version: 1.80.1 (376290515 2024-07-16)

--- a/README.md
+++ b/README.md
@@ -1,9 +1,9 @@
-# **godam** 
+# **godam**
 
 ![Rust 1.81](https://img.shields.io/badge/rust-1.81-orange)
 [![Build Status](https://github.com/nilsiker/godam/actions/workflows/release.yml/badge.svg)](https://github.com/nilsiker/godam/actions/workflows/release.yml)
 
-**godam** (**God**ot **A**sset **M**anager) is an unofficial, lightweight command-line tool designed to streamline the management of assets from the Godot Asset Library. 
+**godam** (**God**ot **A**sset **M**anager) is an unofficial, lightweight command-line tool designed to streamline the management of assets from the Godot Asset Library.
 
 <p align="center">
   <img alt="an icon of a orange pixel-art robot head, with a command line icon obscuring its face, titled 'godam' just below it" src="media/godam.png" />
@@ -24,7 +24,6 @@ Here's a rundown of what **godam** can currently do:
 - **List** all assets managed by **godam**.
 - **Clean** the local asset cache, removing all downloaded zip archives.
 
-
 <p align="center">
   <img alt="command-line showcase of initializing, searching and installing using godam" src="media/godam_basic_example.gif" />
 </p>
@@ -39,7 +38,7 @@ When you run `godam install <ID>`, the following happens:
 2. The `.godam` cache is checked for the asset; if not cached, the asset zip is downloaded.
 3. **godam** maps the asset's ID to its install location in `godam.toml`, keeping track of what plugin ID maps to what install folder.
 4. The asset's `addons` folder is extracted from the zip and copied into your project.
-  
+
 This process is repeated for every asset listed in the `godam.toml` file.
 
 > ⚠️ **godam** currently only respects addon folder structure, meaning that it expects to find the `addons` folder in either directly inside the zip file or located under an immediate folder in the archive. Any other folder structure will throw an error, and cannot be installed using godam.
@@ -50,8 +49,10 @@ Let's walk through installing [LimboAI for Godot 4.3](https://godotengine.org/as
 
 First off you need to install godam. You can download the [latest release binary](https://github.com/nilsiker/godam/releases/latest) and add it to your PATH, or build it from source using `cargo install godam`.
 
-Following this, follow the procedure below to try it out:
-  
+> ⚠️ When installing from source, the MSRV (Minimum Supported Rust Version) is 1.81
+
+Once installed, follow the procedure below to try it out:
+
 1. Navigate to your Godot project directory:  
    `cd path/to/your/godot/project`
 
@@ -66,10 +67,10 @@ Following this, follow the procedure below to try it out:
 
 5. List all assets managed by **godam**:  
    `godam list`
-  
+
 6. Install all assets defined in `godam.toml`:  
    `godam install`
-  
+
 7. Clean the cache:  
    `godam clean`
 
@@ -86,7 +87,7 @@ Here’s what’s on the roadmap for future development:
 - Asset version freezing and upgrading (awaiting asset library API support)
 - Nicer console output
 - Global caching and symlinks
-- Test suites for preventing bugs and regressions 
+- Test suites for preventing bugs and regressions
 - General codebase improvements, improving clarity, maintainability and performance
 
 ---

--- a/README.md
+++ b/README.md
@@ -49,7 +49,7 @@ Let's walk through installing [LimboAI for Godot 4.3](https://godotengine.org/as
 
 First off you need to install godam. You can download the [latest release binary](https://github.com/nilsiker/godam/releases/latest) and add it to your PATH, or build it from source using `cargo install godam`.
 
-> ⚠️ When installing from source, the MSRV (Minimum Supported Rust Version) is 1.81
+> ⚠️ When installing from source, the MSRV (Minimum Supported Rust Version) is 1.81.
 
 Once installed, follow the procedure below to try it out:
 

